q2studio/api/types.py

```diff
@@ -31,3 +31,9 @@ def is_subtype():
             else:
                 nays.append(repr(a))
     return jsonify({'yes': yes, 'no': no})
+
+
+@types.route('/importabletypes', methods=['POST'])
+def get_importable_types():
+    ret = [str(t) for t in PLUGIN_MANAGER.importable_types]
```

Comment:
nitpicky, but `repr` is really what you want instead of `str`, they don't *do* different things, but they kind of *mean* different things (semantic types don't document this distinction, but it's pretty much what our interfaces assume they can use)